modules/mobile/docs/openapi.yaml

```diff
@@ -1,52 +1,358 @@
 openapi: "3.0.0"
 info:
-  version: 1.0.0
+  version: 0.0.0
   title: VA Mobile API
   license:
     name: MIT
 servers:
-  - url: http://api.va.gov
+  - url: http://api.va.gov/mobile
 paths:
-  /mobile:
+  /:
     get:
-      summary: /mobile
-      description: Currently returns a welcome message. In the future given an app version it will return a map of services.
-      operationId: discovery
+      summary: /
+      description: Returns a welcome message.
+      operationId: welcome
       responses:
         '200':
           description: OK
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/Message"
-  /mobile/v0/user:
+
+
+  /v0/user:
     get:
-      summary: /mobile/v0/user
-      description: Returns the user profile
-      operationId: get_user
+      summary: /v0/user
+      description: >
+        \[Partially implemented\] Returns the user profile, including the user's addresses and the services
+        the user has the requisite ids to acccess. Meta data for this endpoint returns all the services available in the API.
+      operationId: user.getProfile
       security:
-        - Bearer: [ ]
+        - Bearer: []
       responses:
         '200':
           description: OK
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/User"
+                properties:
+                  data:
+                    $ref: "#/components/schemas/User"
+                  meta:
+                    properties:
+                      availableServices:
+                        type: array
+                        example:
+                          - appeals
+                          - appointments
+                          - claims
+                          - directDepositBenefits
+                          - letters
+                          - militaryServiceHistory
+                          - userProfileUpdate
+        '401':
```

Comment:
Looked through the OpenAPI docs/stack overflow and there doesn't seem to be a way to reuse these error codes (you can just ref the schema as below)